frontend/src/metabase/new_model/containers/NewModelOptions.jsx

```diff
@@ -8,21 +8,21 @@ import { t } from "ttag";
 
 import { Grid } from "metabase/components/Grid";
 
-import NewQueryOption from "metabase/new_query/components/NewQueryOption";
+import NewModelOption from "metabase/new_model/components/NewModelOption";
 import NoDatabasesEmptyState from "metabase/reference/databases/NoDatabasesEmptyState";
 
 import * as Urls from "metabase/lib/urls";
 
 import {
   getHasDataAccess,
   getHasNativeWrite,
-} from "metabase/new_query/selectors";
+} from "metabase/new_model/selectors";
```

Comment:
Should we use the existing `getHasDataAccess` and `getHasNativeWrite` defined in `metabase/nav/selectors` instead of duplicating them? Maybe it'd be worth extracting them somewhere else instead of `metabase/nav` as they're pretty helpful outside the nav too

Reply:
Yes, that makes sense to me as well. 

Any idea of a place off the top of your head, @kulyk ?

Reply:
Maybe add a new file to `metabase/selectors`? E.g. `metabase/selectors/data`

Reply:
Yep.

I thought of adding these to `metabase/selectors/user` as it's the user's access to data.

Or calling it `metabase/selectors/data-acess` or something.

Went with `metabase/selectors/data`, thank you.

Reply:
@gusaiani, should we use `metabase/selectors/data` in the new model flow as well? So we can remove `metabase/new_model/selectors`

Reply:
Absolutely. Updated.